public/global.css

```diff
@@ -1,63 +1,61 @@
-html, body {
-	position: relative;
-	width: 100%;
-	height: 100%;
+:root {
+  --font-regular: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  --font-monospace: "Courier New", Courier, monospace;
```

Comment:
These are CSS variables. They're pretty handy for stuff like theming and colors. I slapped the font families here because they're different it a couple places, and typing all these font names out multiple times succkkksssss.